tests/math/test_functions.py

```diff
@@ -1362,14 +1452,14 @@ def test_array(self):
         x = np.array(self.x, requires_grad=True)
         y = np.array(self.y, requires_grad=True)
 
-        def cost(weights):
+        def cost(*weights):

```

Comment:
How come this test was changed?

Reply:
While `x` and `y` are markes as trainable above, the list `[x, y]` of course is not. This triggered the warning that with Autograd, soon parameters will need to be marked as trainable explicitly. I'm aware that this is in an intermediate state across the codebase currently, but I thought it might be nice to change it here already, removing warnings from the test suite :)